src/VscPreviewExtension/Microsoft.DocAsCode.Dfm.VscPreview/PreviewCommand.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DocAsCode.Dfm.VscPreview
+{
+    using System.Collections.Generic;
+    using System.IO;
+
+    using Microsoft.DocAsCode.Common;
+
+    public class PreviewCommand
+    {
+        public static PreviewJsonConfig ParsePreviewCommand(string basedir)
+        {
+            string configFilePath = Path.Combine(basedir, PreviewConstants.ConfigFile);
+            PreviewJsonConfig config = new PreviewJsonConfig();
+            if (!string.IsNullOrEmpty(PreviewConstants.ConfigFile) && File.Exists(configFilePath))
+                config = JsonUtility.Deserialize<PreviewJsonConfig>(configFilePath);
+            return MergeDefaultConfig(config);
+        }
+
+        private static PreviewJsonConfig MergeDefaultConfig(PreviewJsonConfig config)
+        {
+            if (string.IsNullOrEmpty(config.MarkupResultLocation))
+                config.MarkupResultLocation = PreviewConstants.MarkUpResultLocation;
+
+            if (string.IsNullOrEmpty(config.Port))
+                config.Port = PreviewConstants.Port;
+
+            if (string.IsNullOrEmpty(config.OutputFolder))
+                config.OutputFolder = PreviewConstants.OutPutFolder;
+
+            if (config.Reference == null)
+                config.Reference = new Dictionary<string, string>()
+                {
+                    {"link", "href"},
+                    {"script", "src"},
+                    {"img", "src"}
+                };
```

Comment:
suggest: add braces for all `if`

---
In reply to: [101431934](https://github.com/dotnet/docfx/pull/1313#discussion_r101431934) [](ancestors = 101431934)